client/signup/steps/site-type/index.jsx

```diff
@@ -0,0 +1,86 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import StepWrapper from 'signup/step-wrapper';
+import SignupActions from 'lib/signup/actions';
+import Button from 'components/button';
+
+import { setSiteType } from 'state/signup/steps/site-type/actions';
+
+import { translate } from 'i18n-calypso';
+
+const SiteTypeStep = React.createClass( {
+	propTypes: {
+		flowName: PropTypes.string,
+		goToNextStep: PropTypes.func.isRequired,
+		positionInFlow: PropTypes.number,
+		setSiteType: PropTypes.func.isRequired,
+		signupProgress: PropTypes.array,
+		stepName: PropTypes.string,
+	},
+
+	submitSiteTypeStep( siteType ) {
+		this.props.setSiteType( siteType );
+
+		SignupActions.submitSignupStep( {
+			processingMessage: translate( 'Setting up your site' ),
+			stepName: this.props.stepName,
+			siteType
+		}, [], { siteType } );
+
+		this.props.goToNextStep();
+	},
+
+    submitBusiness() {
+        this.submitSiteTypeStep( "business" );
+    },
+
+    submitPersonal() {
+        this.submitSiteTypeStep( "personal" );
+    },
+
+	skipStep() {
+		this.submitSiteTypeStep( '' );
+	},
+
+	renderSiteTypeStep() {
+		return (
+			<div className="site-type__wrapper">
+				<Button onClick={ this.submitBusiness } primary>Business</Button>
+                <Button onClick={ this.submitPersonal } primary>Personal</Button>
+			</div>
+		);
+	},
+	render() {
+		const headerText = translate( 'Welcome to WordPress.' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **82** times:
`translate( 'Welcome to WordPress!' )` *ES Score: 15*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).